scripts/patch_metal_spec.py

```diff
@@ -197,7 +202,7 @@ def loadYaml(fn):
 # FIX 7. mitigate bug in openapi-generator
 # https://github.com/OpenAPITools/openapi-generator/issues/14780
 
-del fixedSpec['components']['schemas']['Plan']['properties']['deployment_types']['uniqueItems']
+#del fixedSpec['components']['schemas']['Plan']['properties']['deployment_types']['uniqueItems']
```

Comment:
If this patch is no longer needed we should delete it instead of commenting it out.

Reply:
I'll remove it.